infra/modules/gcp/main.tf

```diff
@@ -160,14 +160,6 @@ resource "google_project_iam_custom_role" "bucket-write" {
   permissions = ["storage.objects.create", "storage.objects.delete"]
 }
 
-output "salt_secret_id" {
-  value = google_secret_manager_secret.pseudonymization-salt.secret_id
-}
-
-output "salt_secret_version_number" {
-  value = trimprefix(google_secret_manager_secret_version.initial_version.name, "${google_secret_manager_secret.pseudonymization-salt.name}/versions/")
-}
```

Comment:
also potentially breaking change, right? could maintain this to keep interface consistent (mark as deprecated, and we can remove in in `v0.5.x`